modules/ROOT/pages/flex-gateway-replicas.adoc

```diff
@@ -0,0 +1,25 @@
+= Replicas
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+A replica is an instance of the Flex Gateway. By default, Flex Gateway runs a single replica. Running multiple replicas enables scaling, load balancing, and high availability.
+
+When using Anypoint Flex Gateway to manage APIs in production, you should at least have more than one replica to avoid a single point of failure. Running multiple replicas also increases performance if your API service receives a significant amount of traffic.
```

Comment:
**you should at least have** more than... > **set** more than...

Reply:
I think "run" could be the most appropriate.

Reply:
applied (run more than one replica)